### PR TITLE
Run view tabs

### LINKED ISSUE #285: Refactor view/edit metadata based on new mockups
Based on the latest UI revisions suggested by Kristina, we will be moving the current view/edit tale metadata functionality from the browse page to the Run page for a tale instance.

Based on the [10/31 planning meeting](https://wholetale.readthedocs.io/development/meetings/2018-10-31.html#edit-view-tale-metadata) this includes:
* Adding "Metadata" tab to Run page
* Refactor view/edit page per mockup
* Out of scope for this issue: publications, browse data, license, derived from, versions, dataset citation information (requires backend changes)
* If user has edit access, they should be able to modify:
  * Title, authors, categories, description, image, change environment
* The following fields are read-only: 
  * Datasets used is derived from current state of data
  * DOI-level citation or other human-readable format for top-level registered item

See also:
* Mockups  for redesign ["Metadata" panel](http://www.crc.nd.edu/~kfurse/wholetale-css-mock-up-sui_f2018/run-meta.html)
* [Invision flow](https://projects.invisionapp.com/freehand/document/nPpPcHSRg) Fall 2018 revisions section

### LINKED ISSUE #302: Add tab component to Run view
Add the "tab" component to the Run view:

Part of this work may already be complete based on https://github.com/whole-tale/dashboard/pull/297

### LINKED ISSUE #303: Add the "Interact" tab with current iFrame/popup
Note, this isn't fully reflected in the [mockup](http://www.crc.nd.edu/~kfurse/wholetale-css-mock-up-sui_f2018/run.html).

Goal: 
* The user can select the Run > Interact tab to display the iframed tale (if iframe is supported), otherwise the usual message is displayed. From the interact tab, the user can pop-out the instance to a new window or access context-sensitive help, as is supported today.

Completion criteria:
* [ ] "Interact" tab displays iframed tale or current messages if no instance selected or iframe not supported
* [ ] Documentation updated
* [ ] Manual test case provided

### PR DESCRIPTION
**Pull request by cyberdante**
This PR includes @bodom0015's previous work, but instead of merging into master, I rebased that work into my branch so we can keep this contained.

Fixes #302 
Fixes #285 
Fixes #303

### CODE REVIEW

**Comment by cyberdante** [comment]
Asking for extra pairs of eyes to check this. Thanks in advance!

**Comment by Xarthisius** [comment]
Is the "Files" tab a stub right now or did it break?

File: config/environment.js [hunk 1]
```diff
@@ -82,10 +82,10 @@ module.exports = function (environment) {
...
 
   if (environment === 'production') {
-    ENV.apiHost = 'apiHOST';
-    ENV.wholeTaleHost = 'https://dashboardHOST';
-    ENV.dataOneHost = 'dataOneHOST';
-    ENV.authProvider = 'authPROVIDER';
+    ENV.apiHost = 'https://girder.dev.wholetale.org';//'apiHOST';
+    ENV.wholeTaleHost = 'https://dashboard.dev.wholetale.org';//'https://dashboardHOST';
+    ENV.dataOneHost = 'https://dev.nceas.ucsb.edu/knb/d1/mn/v2';//'dataOneHOST';
+    ENV.authProvider = 'Globus';//'authPROVIDER';
```
Comment (Xarthisius) [inline config/environment.js]: I think `production` should stay `production` ;-) Could you please revert that?

Comment (cyberdante) [reply config/environment.js]: Yes, sorry about that. It lingered from testing some other changes.

Comment (cyberdante) [reply config/environment.js]: Fixed with commit [#0a485c9](https://github.com/whole-tale/dashboard/pull/325/commits/0a485c90c0225bd464c6c278322f37f193f5e5e9)

**Comment by cyberdante** [comment]
Thanks for the feedback, @ThomasThelen. As the commit message specifies, I'm just matching the comps: `http://www.crc.nd.edu/~kfurse/wholetale-css-mock-up-sui_f2018/run-files-home.html` by @kdavis-nd. I will hide the `Publish` functionality from the ellipsis menu for now. The three functions/icons that previously existed in the paddleboard menu were for going fullscreen, opening a tale externally on another tab, and for visiting the read the docs link. All those are currently available we we're not losing any functionality. The content within all tabs (Home, External Data, and Tale Workspace) are currently being worked on. I pushed those changes so they were available for Adam so we didn't work on the same inadvertently. I think that the files going beyond the lower border of their container was a great catch!!! Thanks again. And if I don't hear otherwise from either @craig-willis, @Xarthisius or @kdavis-nd I will comply with your request to use the capitalized, non-underscored names for those folders.

**Comment by abrinckm** [comment]
I need to solve this somehow... 

also, @ThomasThelen @bodom0015 I have a task to implement the external data tab. So I'm thinking to add that work to this pull request. Does that make sense to do?

**Comment by ThomasThelen** [comment]
I think in the spirit of small incremental changes we should merge this PR without adding any more features. Since we don't want to merge into `master` until we tag 0.5 I think something like creating a feature branch off of this PR and rebasing with `master` once the PR is merged would work.

**Comment by abrinckm** [comment]
Okay I will branch from this PR

**Comment by cyberdante** [comment]
@Xarthisius - I still need to know what to do with this branch after `test_325`

**Comment by Xarthisius** [comment]
Obsoleted by #335

**Pull request closed without merging**